src/solver/rosenbrock.cu

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2023 National Center for Atmospheric Research
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+#include <chrono>
+#include <vector>
+#include <iostream>
+#include <micm/util/cuda_param.hpp>
+
+namespace micm{
+    namespace cuda{
+        __global__ void AlphaMinusJacobianKernel(size_t n_grids,
+                                                double* d_jacobian,
+                                                size_t* d_jacobian_diagonal_elements,
+                                                size_t jacobian_diagonal_elements_size,
+                                                double alpha)
```

Comment:
I think `alpha` is defined as a constant for a given Rosenbrock solver. Could it be a global constant variable instead of an argument?

Reply:
I think `alpha` here is the `gamma` parameter in my paper (Equation 2 of https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2016MS000863). Is it correct?

Reply:
@sjsprecious almost, it's 1/(H*gamma)

https://github.com/NCAR/micm/blob/main/include/micm/solver/rosenbrock.inl#L430

Reply:
Thanks @K20shores . I see the difference now.